src/fragments/lib/storage/js/get-properties.mdx

```diff
@@ -0,0 +1,49 @@
+Get the properties of the file using `getProperties` method.
+
+```typescript
+const key: string = 'srcKey';
+await Storage.getProperties(key);
```

Comment:
Looking at the Flutter page, it might be helpful to demonstrate using the returned content in some way as part of the example. The Flutter example is just printing the file size, maybe something like printing the size and last updated and catching/printing something unique when the file accessed isn't present?